modules/db/models/wallet.js

```diff
@@ -94,6 +96,20 @@ module.exports = (osseus) => {
     })
   }
 
+  wallet.updateBalances = (condition, blockchainAmount, offchainAmount) => {
+    return new Promise((resolve, reject) => {
+      const update = {}
+      if (blockchainAmount) update['balances.$.blockchainAmount'] = blockchainAmount
+      if (offchainAmount) update['balances.$.offchainAmount'] = offchainAmount
```

Comment:
remove offchain update balance
offchain balance should update only with a valid transaction